azure-pipelines.yml

```diff
@@ -44,14 +44,7 @@ steps:
       python -m pytest -xv --cov pymc4 --junitxml=junit/test-results.xml --cov-report xml --cov-report term --cov-report html .
     displayName: "pytest"
 
-  - task: PublishTestResults@2
+  - script: |
+      bash <(curl -s https://codecov.io/bash)
```

Comment:
neat! does this just run a bash script that will check for $CODECOV_TOKEN?

Reply:
Yup. At least it doesn't seem to be required on [azure](https://docs.codecov.io/docs/about-the-codecov-bash-uploader)